### PR TITLE
Feature/vudrm 1800/inital creation

### PR DESCRIPTION
**Pull request by bengo1990**
Working Agnoplay example.

### CODE REVIEW

**Review by fschurer — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,70 @@
...
+
+1. Open the repository in your favourite javascript editor.
+2. In the file `agnoplay_vudrm.js` replace `<your-vudrm-token>` with a VUDRM token from [https://admin.vudrm.tech](https://admin.vudrm.tech).
+3. In the file `agnoplay_vudrm.js` replace `<brand-name>` with the brand name given to you by Agnoplay.
+4. In the file `agnoplay_vudrm.js` replace `<agnoplay-license-key>` with your Agnoplay key.
+5. In the file `agnoplay_vudrm.js` replace `<video-file-url>` with your stream URL.
+6. In the file `agnoplay_vudrm.js` replace `<video-title>` with your video name.
+7. In the file `agnoplay_vudrm.js` replace `<video-poster>` with a link to your thumbnail image.
+8. In the file `index.html` replace `<yourdomain>` on line 18 with your brand name. For example, for Vualto, this will be `https://player.vualto.com/static/agnoplay/js/agnoplay.js`.
+9. Add the host `agnoplay.drm.technology` to your local machine's hosts file.
```
Comment (somerobmitchell) [inline README.md:31]: use `vudrm.tech`

File: Gruntfile.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                    ]
+                }
+            }
+        },
+
+        connect: {
+            server: {
+                options: {
+                    protocol: "https",
+                    hostname: "agnoplay.drm.technology",
```
Comment (somerobmitchell) [inline Gruntfile.js:57]: use `vudrm.tech`

**Review by somerobmitchell — DISMISSED** [review]

File: Gruntfile.js (hunk 2 above)
Comment (bengo1990) [reply Gruntfile.js:57]: Actioned.

File: README.md (hunk 1 above)
Comment (bengo1990) [reply README.md:31]: Actioned.

**Review by fschurer — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,2 +1,70 @@
...
+2. In the file `agnoplay_vudrm.js` replace `<your-vudrm-token>` with a VUDRM token from [https://admin.vudrm.tech](https://admin.vudrm.tech).
+3. In the file `agnoplay_vudrm.js` replace `<brand-name>` with the brand name given to you by Agnoplay.
+4. In the file `agnoplay_vudrm.js` replace `<agnoplay-license-key>` with your Agnoplay key.
+5. In the file `agnoplay_vudrm.js` replace `<video-file-url>` with your stream URL.
+6. In the file `agnoplay_vudrm.js` replace `<video-title>` with your video name.
+7. In the file `agnoplay_vudrm.js` replace `<video-poster>` with a link to your thumbnail image.
+8. In the file `index.html` replace `<yourdomain>` on line 18 with your brand name. For example, for Vualto, this will be `https://player.vualto.com/static/agnoplay/js/agnoplay.js`.
+9. Add the host `agnoplay.vudrm.tech` to your local machine's hosts file.
+10. Run the command `grunt serve`.
+11. Load a supported browser and go to `https://agnoplay.drm.technology:14703`.
```
Comment (somerobmitchell) [inline README.md:33]: needs changing to vudrm.tech

**Review by somerobmitchell — DISMISSED** [review]

File: README.md (hunk 3 above)
Comment (bengo1990) [reply README.md]: Actioned.

**Review by somerobmitchell — APPROVED** [review]

**Pull request merged**